src/primitive_core.cc

```diff
@@ -445,6 +445,7 @@ PRIMITIVE(read_int_little_endian) {
 
 PRIMITIVE(args) {
   char** argv = process->args();
+#ifndef TOIT_FREERTOS
```

Comment:
This feels like a completely unrelated change.
If we don't want to have compile-time arguments then we should remove them in a different PR.

Reply:
Change removed.